target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -131,7 +131,8 @@ ath79_setup_interfaces()
 			"0@eth0" "1:lan" "2:lan" "3:lan" "4:lan"
 		;;
 	comfast,cf-wr650ac-v1|\
-	comfast,cf-wr650ac-v2)
+	comfast,cf-wr650ac-v2|\
+	zyxel,nbg6616)
```

Comment:
Does this have a label MAC address (possibly $(mtd_get_mac_ascii u-boot-env ethaddr) ) that can be set via label_mac in the lower part of this file?

Reply:
all mac addresses are printed on the label (2ghz, 5ghz, wan, lan). which one should be set as label_mac in this case?

Reply:
Okay, if the "base address" (i.e. the non-incremented one) is the first on the label, then you could use that one. Or you just leave it as it is. Both options are possible.

Reply:
the first one is indeed that base address stored in uboot-env, added it. label_mac for nbg6716 is still missing afaik.

Reply:
Yes, because nobody cared during support of the nbg6716, and the label MAC shouldn't be guessed.